rajawali/src/androidTest/java/org/rajawali3d/animation/SlerpAnimation3Dtest.java

```diff
@@ -0,0 +1,40 @@
+package org.rajawali3d.animation;
+
+import android.test.suitebuilder.annotation.SmallTest;
+import static org.junit.Assert.*;
+import org.junit.*;
+
+import org.rajawali3d.animation.SlerpAnimation3D;
+import org.rajawali3d.math.vector.Vector3;
+import org.rajawali3d.math.Quaternion;
+
+
+/**
+ * @author Jared Woolston (jwoolston@keywcorp.com)
```

Comment:
Did he write this class though? :thinking:

Reply:
Well he wrote most of it...
I just copied his file and modified a couple of lines.